html/changelog.html

```diff
@@ -51,6 +51,20 @@
 		
 		-->
 		<div class="commit sansserif">
+			<h2 class="date">24 August 2020</h2>
+			<h3 class="author">Jared-Fogle updated:</h3>
+			<ul class="changes bgimages16">
+				<li class="bugfix">Being polled as a ghost for things like imaginary friends will now show the player's real name, rather than their displayed name.</li>
+				<li class="bugfix">You can now sign your name on paper again. Remember: %s.</li>
+			</ul>
+			<h3 class="author">tralezab updated:</h3>
+			<ul class="changes bgimages16">
+				<li class="tweak">when admins change the emergency shuttle timer, it gives a way more accurate estimate.</li>
+			</ul>
+			<h3 class="author">uomo91 updated:</h3>
+			<ul class="changes bgimages16">
+				<li class="tweak">Toolbelts can now hold pipe painters.</li>
+				<li class="bugfix">Bookshelves do not show shelves inside anymore when you crowbar them out.</li>
```

Comment:
Deactivate actions in your github. Ping me in the coder channel on discord if you need to know how.
Then revert the commit adding this.